Makefile

```diff
@@ -144,28 +33,4 @@ clean-docker-images:
 
 clean-docker-all: clean-docker-containers clean-docker-buildcache clean-docker-images
 
-clean-all: clean-pants-cache clean-docker-buildcache clean-docker-containers
-
-
-
-########## CI targets #################
-
-ci-setup:
-	pants --version  # Bootstrap Pants.
-
-ci-lint: ci-setup
-	# this target uses git to sort out only running on changes since the last main.
-	pants --changed-since=origin/main \
-	update-build-files --check \
-	lint
-
-ci-fmt: ci-lint
-	pants --changed-since=origin/main fmt
-
-ci-tests:
-	@echo "running tests with parametrized platform: $(PARAMETRIZE)"
-	pants test lumigator/python/mzai/backend/backend/tests:backend_tests@parametrize=$(PARAMETRIZE)
-	pants test lumigator/python/mzai/sdk/tests:sdk_tests@parametrize=$(PARAMETRIZE)
-
-ci-publish-images:
-	pants package publish lumigator/python/mzai/backend:lumigator
+clean-all: clean-docker-buildcache clean-docker-containers
```

Comment:
should we consolidate clean-docker-all and clean-all at this point?

Reply:
That makes sense, we might want to add a uv-clean or something too